src/sidebar/app.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var queryString = require('query-string');
+var addAnalytics = require('./ga');
```

Comment:
Nit. Local modules should be grouped separately from npm dependencies:

```
var thing = require('npm-dep');
var thing2 = require('npm-dep2');

var localThing = require('./local-module');
```

Our Python code follows the same convention.